README.md

```diff
@@ -47,11 +47,12 @@ Services supporting ActivityPub federation.
 * [ActivityPub for WordPress](https://wordpress.org/plugins/activitypub/) - An ActivityPub plugin for WordPress.
 * [Mobilizon](https://joinmobilizon.org/en/) - A federated organization and mobilization platform.
 * [reel2bits](https://reel2bits.org/) - A Federated audio and podcasts sharing platform.
-* [Socialhome](https://socialhome.network/) - A federated personal profile with social networking functionality. 
```

Comment:
Should SocialHome be removed?

Reply:
No. Sorry, I was doing some lazy editing in my web-editor.